client/src/services/requests.ts

```diff
@@ -0,0 +1,8 @@
+const options = {
+  method: "GET",
+  headers: {
+    accept: "application/json",
+    Authorization:
+      "Bearer eyJhbGciOiJIUzI1NiJ9.eyJhdWQiOiJmMTdmM2Y0ODZmMDFhNzZhNGUzMjJlZmI5NTExMmRlZiIsIm5iZiI6MTczNjQxMjIzMS4wMTQwMDAyLCJzdWIiOiI2NzdmOGM0N2M4MWFjYWE2M2RiYjNjYTkiLCJzY29wZXMiOlsiYXBpX3JlYWQiXSwidmVyc2lvbiI6MX0.-wSwEdNS3XemAnW1ZYt0qj1nlil4ccn-w-2MPmVk4Xs",
```

Comment:
Warning :)
Ne pas mettre les clés API en dur dans le code :)
Mettre la clé API dans le .env

Documentation 

https://vite.dev/guide/env-and-mode